core/main/src/broker/endpoint_broker.rs

```diff
@@ -1648,12 +1650,19 @@ impl BrokerOutputForwarder {
         } else {
             None
         };
-        let parse_result = serde_json::from_slice::<Value>(data);
-        debug!("parse result {:?}", parse_result);
-        if parse_result.is_err() {
-            return Err(RippleError::ParseError);
-        }
-        let result = Some(parse_result.unwrap());
+
```

Comment:
Only validate the response body if it exists, empty bodies are valid for both GET and POST methods.